### ISSUE TITLE
Skip to Content not visible for keyboard-only users

### ISSUE THREAD

**Issue by mgifford** [ckan/ckan#7194]
**CKAN version**
https://demo.ckan.org/

**Describe the bug**

Skiplink exists but is not focusable:
`<div class="sr-only sr-only-focusable"><a href="#content">Skip to content</a></div>
`
Not all keyboard-only users use screen readers. All links should be visible on focus for keyboard only users. 

**Steps to reproduce**
Tab into the site and you do not see the "Skip to Content" text.

**Expected behavior**
It should be visible. [Drupal](https://www.drupal.org/) is a good example to follow.

**Additional details**
"Skip to Main Content" is probably better than "Skip to Content" 

Similar to https://github.com/ckan/ckan/issues/5733#issuecomment-1307428818 you should be able to use Bootstrap to help accomplish this:

https://getbootstrap.com/docs/5.0/helpers/visually-hidden/
> Use .visually-hidden-focusable to visually hide an element by default, but to display it when it’s focused (e.g. by a keyboard-only user)
